docs/source/deployorderer/ordererchecklist.md

```diff
@@ -0,0 +1,320 @@
+        # is enabled. ClientAuthRequired requires client certificate authentication
+        # at the TLS layer to access all resources.
+        ClientAuthRequired: false
+
+        # Paths to PEM encoded ca certificates to trust for client authentication
+        ClientRootCAs: []
+```
+
+* **`ListenAddress`**: (required when using the operations service) Specify the address and port of the operations server.
+* **`Enabled`**: (required when using the operations service) Must be `true` if the operations service is being used.
+* **`Certificate`**: (required when using the operations service) Can be the same file as the `General.TLS.Certificate`.
+* **`PrivateKey`**: (required when using the operations service) Can be the same file as the `General.TLS.PrivateKey`.
+* **`ClientAuthRequired`**: (required when using the operations service) Must be set to `true` to enable mutual TLS between the client and the server.
+* **`ClientRootCAs`**: (required when using the operations service) Similar to the client root CA cert file in TLS, it contains a list of client root CA certificates that can be used to verify client certificates. If the client enrolled with the orderer organization CA, then this value is the orderer organization root CA cert.
+
+## Metrics.*
+
+By default this is disabled, but if you want to monitor the metrics for the orderer, you need to use `StatsD` as your metric provider. `StatsD` uses a "push" model, pushing metrics from the orering node to a `StatsD` endpoint. Because of this, it does not require configuration of the operations service itself. See the list of [Available metrics for the orderer](../metrics_reference.html#orderer-metrics).
+
+Because Prometheus utilizes a "pull" model there is not any configuration required, beyond making the operations service available. Rather, Prometheus will send requests to the operations URL to poll for available metrics.
+
+```
+    # The metrics provider is one of statsd, prometheus, or disabled
+    Provider: disabled
+
+    # The statsd configuration
+    Statsd:
+      # network type: tcp or udp
+      Network: udp
+
+      # the statsd server address
+      Address: 127.0.0.1:8125
+
+      # The interval at which locally cached counters and gauges are pushed
+      # to statsd; timings are pushed immediately
+      WriteInterval: 30s
+
+      # The prefix is prepended to all emitted statsd metrics
+      Prefix:
+```
+
+* **`Provider`**: Must be `enabled` to use `StatsD` or `Prometheus` metrics for the ordering node.
+* **`Statsd.Address`**: (required to use `StatsD` metrics for the ordering node) When `StatsD` is enabled, you will need to configure the `hostname` and `port` of the `StatsD` server so that the ordering node can push metric updates.
+
+## Consensus.*
+
+These values could vary depending on consensus plugin. For instance, a BFT plugin would very likely have a different set of options here.
```

Comment:
I think this is my text verbatim, but I think perhaps it might make more sense to say something like:

"The values of this section vary by consensus plugin.  If you are using the recommended etcdraft consensus plugin, see the following documentation.  If you are using a different consensus plugin, refer to its documentation for allowed keys and recommended values.